src/app/layout/header/header.component.ts

```diff
@@ -301,13 +298,24 @@ export class HeaderComponent implements OnInit, OnDestroy {
             if (this.notif.pendingSyncCount < 0) {
                 this.notif.pendingSyncCount = 0;
             }
-        }
-        catch (e) {
+            this._initZeroCheckerForPendingSyncCount();
+        } catch (e) {
             throw e;
         }
     }
 
-    private async _initAssetsOfferedToMeSyncCount(){
+    private _initZeroCheckerForPendingSyncCount(): void {
+        this.notifService.pendingSyncCount$
+            .pipe(takeUntil(this._subscription$))
```

Comment:
You can add here filter operator, i mean `filter(count => count === 0)`. That way subscribe doesn't need if statement